environment.sh

```diff
@@ -7,7 +7,7 @@ PROMETHEUS_NAME_PROXY="prometheus_proxy"
 
 PROMETHEUS_PORT=9090
 
-export PORT=8080
+export PORT=":8080"
```

Comment:
Instead of port we could rename this to ADDRESS

And the variables in https://github.com/bf2fc6cc711aee1a0c2a/observability-remote-write-proxy/blob/1eeb5ba57ea12312c640298c8495670ee99f27b3/main.go#L18 as well.

Reply:
Saying that because a port is usually a number, so seeing `:` in there makes it look very confusing.
What do you think? @valeriiashapoval @pb82

Reply:
makes sense @machi1990, will add the change now. Just added the fix due to the script not behaving properly (had issues with it on Friday)

Reply:
thank you!

Reply:
we will address this in a follow up PR